### PR TITLE
Adapt GUI to render book requests

### PR DESCRIPTION
**Pull request by yl-ang**
- Update GUI to render the requesters for the book if any
- Add logging for BookList and PatronList panels (For tP task)
- Update PPP

### CODE REVIEW

File: src/main/java/seedu/address/ui/BookCard.java [hunk 1]
```diff
@@ -91,7 +98,10 @@ public void setAuthor(Book book) {
                     .map(Author::toString)
                     .collect(Collectors.joining(", "));
             authors.setText("Author: " + authorsString);
+            return;
         }
+        authors.setVisible(false);
+        authors.setManaged(false);
```
Comment (bingsen0806) [inline src/main/java/seedu/address/ui/BookCard.java:104]: Based on the principle of "Making the happy path prominent", I think you should include the last two lines (103-104) in the if statement, and move the current lines in the if statement outside of the if statement

File: src/main/java/seedu/address/ui/BookListPanel.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
      */
     public BookListPanel(ObservableList<Book> bookList) {
         super(FXML);
+        logger.info("Creating Book List Panel");
```
Comment (bingsen0806) [inline src/main/java/seedu/address/ui/BookListPanel.java:28]: Good use of logging!

File: src/main/resources/view/BookListCard.fxml [hunk 3]
```diff
@@ -36,6 +36,9 @@
       <!-- Ensures that the borrower and book return date line will not be rendered if the book is not borrowed-->
       <Label fx:id="borrower" styleClass="cell_small_label" visible="false" managed="false" wrapText="true" />
       <Label fx:id="bookReturnDate" styleClass="cell_small_label" visible="false" managed="false" wrapText="true" />
+      <!-- Ensures that the requesters label will not be rendered if the book is not requested by anyone-->
+      <FlowPane fx:id="bookRequestTag" />
+      <Label fx:id="requestedBy" styleClass="cell_small_label" visible="false" managed="false" wrapText="true" />
```
Comment (bingsen0806) [inline src/main/resources/view/BookListCard.fxml:41]: Good attempt to set the label to not visible initially!

**Review by bingsen0806 — APPROVED** [review]
LGTM! Feel free to take a look at the minor code quality improvements as suggested.

File: src/main/java/seedu/address/ui/BookCard.java (hunk 1 above)
Comment (yl-ang) [reply src/main/java/seedu/address/ui/BookCard.java:104]: Thanks for pointing out, will update in the upcoming PR 👍

**Pull request merged**